pkg/morph/deploy/notary.go

```diff
@@ -869,3 +878,357 @@ func makeUnsignedDesignateCommitteeNotaryTx(roleContract *rolemgmt.Contract, com
 
 	return tx, nil
 }
+
+// newCommitteeNotaryActor returns notary.Actor builds and sends Notary service
+// requests witnessed by the specified committee members to the provided
+// Blockchain. Given local account pays for transactions.
+func newCommitteeNotaryActor(b Blockchain, localAcc *wallet.Account, committee keys.PublicKeys) (*notary.Actor, error) {
+	committeeMultiSigM := smartcontract.GetMajorityHonestNodeCount(len(committee))
```

Comment:
you make it twice (one more in the `listenCommitteeNotaryRequests`). can we do it once?

Reply:
as a micro optimization - yes. For me it's not worth it, because the calculations are not so complicated, and you will have to write a lot of preconditions (initialize this account in no other way), and now the signature of this function looks quite concise

Reply:
> you will have to write a lot of preconditions

or duplicate the code. but ok